### PR TITLE
General overall code clean

### PR DESCRIPTION
**Pull request by JasonHughes94**
* Remove a bunch of unused using statements
* Marked fields as readonly where possible
* Updated to use var as standard
* Updated field naming to be standard
* Updated const naming to be standard
* Removed unused local variables
* Remove unused extension
* Updated truncate implementation to be a little cleaner (at least in my opinion)

If you want me to change anything just fire in a comment and I'll take a look tomorrow

### CODE REVIEW

File: Modix.Data.Test/Sandbox.cs [hunk 1]
```diff
@@ -1,27 +1,11 @@
 ﻿using System;
```
Comment (JakenVeina) [inline Modix.Data.Test/Sandbox.cs]: This whole file should just be removed, honestly. It was something I made as an easy entry point to fuck around with some EF queries, early on.

File: Modix.Services/Core/AuthorizationAutoConfigBehavior.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 using Discord;
 using Discord.WebSocket;
+using Modix.Services.Moderation;
```
Comment (JakenVeina) [inline Modix.Services/Core/AuthorizationAutoConfigBehavior.cs]: Why was this added?

File: Modix.Services/Moderation/ModerationService.cs [hunk 3]
```diff
@@ -227,7 +227,7 @@ public async Task UnConfigureGuildAsync(IGuild guild)
             AuthorizationService.RequireAuthenticatedUser();
             AuthorizationService.RequireClaims(AuthorizationClaim.DesignatedRoleMappingDelete);
 
-            using (var transaction = await DesignatedRoleMappingRepository.BeginDeleteTransactionAsync())
```
Comment (JakenVeina) [inline Modix.Services/Moderation/ModerationService.cs]: This is actually a bug. `IRepositoryTransaction`s do not auto-commit. They auto-rollback if you haven't committed. There actually needs to be a call to `transaction.Commit();` at the end of this `using` block.

File: Modix.Services/Moderation/ModerationService.cs [hunk 4]
```diff
@@ -515,7 +515,7 @@ public Task<IReadOnlyCollection<ModerationActionSummary>> SearchModerationAction
 
         private async Task<IRole> GetOrCreateDesignatedMuteRoleAsync(IGuild guild, ulong currentUserId)
         {
-            using (var transaction = await DesignatedRoleMappingRepository.BeginCreateTransactionAsync())
```
Comment (JakenVeina) [inline Modix.Services/Moderation/ModerationService.cs]: Same as above.

File: Modix.Services/Moderation/ModerationService.cs [hunk 5]
```diff
@@ -528,7 +528,7 @@ private async Task<IRole> GetOrCreateDesignatedMuteRoleAsync(IGuild guild, ulong
                     return guild.Roles.First(x => x.Id == mapping.Role.Id);
 
                 var role = guild.Roles.FirstOrDefault(x => x.Name == MuteRoleName)
-                    ?? await guild.CreateRoleAsync(MuteRoleName);
+                           ?? await guild.CreateRoleAsync(MuteRoleName);
```
Comment (JakenVeina) [inline Modix.Services/Moderation/ModerationService.cs]: Why the space here? A continuation of a statement should be indented 1 level deeper than the line on which the statement starts.

**Comment by JakenVeina** [comment]
Good luck with your t-shirt.

File: Modix.Services/Core/AuthorizationAutoConfigBehavior.cs (hunk 2 above)
Comment (JasonHughes94) [reply Modix.Services/Core/AuthorizationAutoConfigBehavior.cs]: This was added due to the comments complaining that it required the reference

File: Modix.Services/Moderation/ModerationService.cs (hunk 3 above)
Comment (JasonHughes94) [reply Modix.Services/Moderation/ModerationService.cs]: I didn't know that

**Comment by JasonHughes94** [comment]
The intention of this PR wasn't to get a t-shirt. If you look back at the commit history I've done this in the past.

I plan on doing another pass through this as there is multiple other sections of code that could be improved/cleaned up.

I also plan of doing a run through of the front end code later in the week.

File: Modix.Services/Core/AuthorizationAutoConfigBehavior.cs (hunk 2 above)
Comment (JakenVeina) [reply Modix.Services/Core/AuthorizationAutoConfigBehavior.cs]: Oh, I see it. Yeah, that's actually copypasta. The comment should be referencing the class itself, `AuthorizationAutoConfigBehavior`.

File: Modix.Services/Moderation/ModerationService.cs (hunk 3 above)
Comment (JakenVeina) [reply Modix.Services/Moderation/ModerationService.cs]: If you want, you can submit that as a bugfix in a separate PR.

**Comment by JakenVeina** [comment]
Just to be clear, I meant no offense. Just a joke about Hacktoberfest. Any commit that provides actual improvement to the codebase is fine by me. And you actually found a bug in the process.

File: Modix.Services/Core/AuthorizationAutoConfigBehavior.cs (hunk 2 above)
Comment (JasonHughes94) [reply Modix.Services/Core/AuthorizationAutoConfigBehavior.cs]: No problemo, I'll make the change for this

File: Modix.Services/Moderation/ModerationService.cs (hunk 3 above)
Comment (JasonHughes94) [reply Modix.Services/Moderation/ModerationService.cs]: Will do

File: Modix.Services/Moderation/ModerationService.cs (hunk 5 above)
Comment (JasonHughes94) [reply Modix.Services/Moderation/ModerationService.cs]: This can be removed, must have hit tab by mistake

**Comment by JasonHughes94** [comment]
It's no problem :D 

I just wanted to put this one in first before I maybe pick up other open issues :)
